lib/src/diff.rs

```diff
@@ -193,30 +193,24 @@ pub(crate) fn unchanged_ranges(
     // the LCS.
     let mut uncommon_shared_words = vec![];
     while !left_histogram.count_to_words.is_empty() && uncommon_shared_words.is_empty() {
+        let left_count = *left_histogram.count_to_words.first_key_value().unwrap().0;
```

Comment:
nit: `.remove_entry()` can be used to extract `(left_count, left_words)` pair.

Reply:
Ah, thanks. And `.pop_first()` is even better.